sdk/storage/azure-storage-blob-stress/src/main/java/com/azure/storage/blob/stress/DownloadContent.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.stress;
+
+import com.azure.core.util.Context;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.storage.blob.BlobAsyncClient;
+import com.azure.storage.blob.BlobClient;
+import com.azure.storage.blob.stress.utils.OriginalContent;
+import com.azure.storage.blob.stress.utils.TelemetryHelper;
+import com.azure.storage.stress.StorageStressOptions;
+import reactor.core.publisher.Mono;
+
+public class DownloadContent extends BlobScenarioBase<StorageStressOptions> {
+    private static final ClientLogger LOGGER = new ClientLogger(DownloadContent.class);
+    private static final TelemetryHelper TELEMETRY_HELPER = new TelemetryHelper(DownloadContent.class);
+    private static final OriginalContent ORIGINAL_CONTENT = new OriginalContent();
+    private final BlobClient syncClient;
+    private final BlobAsyncClient asyncClient;
+    private final BlobAsyncClient asyncNoFaultClient;
+
+    public DownloadContent(StorageStressOptions options) {
+        super(options, TELEMETRY_HELPER);
+        this.asyncNoFaultClient = getAsyncContainerClientNoFault().getBlobAsyncClient(options.getBlobName());
+        this.syncClient = getSyncContainerClient().getBlobClient(options.getBlobName());
+        this.asyncClient = getAsyncContainerClient().getBlobAsyncClient(options.getBlobName());
+    }
+
+    @Override
+    protected boolean runInternal(Context span) {
+        try {
+            return Boolean.TRUE.equals(ORIGINAL_CONTENT.checkMatch(syncClient.downloadContent().toFluxByteBuffer(),
+                span).block());
```

Comment:
```suggestion
            return ORIGINAL_CONTENT.checkMatch(syncClient.downloadContent().toFluxByteBuffer(),
                span).block().booleanValue();
```

Reply:
When I implement your suggestion or remove the Boolean.TRUE.equals, I get the following warning:

**Unboxing of 'ORIGINAL_CONTENT.checkMatch(syncClient.downloadContent().toFluxByteBuffer(), span).block()' may produce 'NullPointerException'**